streams/src/main/java/org/apache/kafka/streams/processor/internals/AbstractTask.java

```diff
@@ -70,6 +71,15 @@ public boolean isClosed() {
         return state;
     }
 
+    @Override
+    public void revive() {
+        if (state == CLOSED) {
+            transitionTo(CREATED);
+        } else {
+            throw new IllegalStateException("Illegal state " + state() + " while committing standby task " + id);
```

Comment:
Remove "standby" from error message (unless this only applies to standbys?)

Reply:
Ack.